docs/source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../predictsignauxfaibles/"))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "predictsignauxfaibles"
+copyright = "2021, Simon Lebastard, Raphaël Ventura, Vincent Viers"
```

Comment:
A verifier mais je ne sais pas si le projet est copyrighté ? Vu qu'il est open-source ?

Reply:
Dans tous les cas je dirais `copyright = signaux-faibles`

Reply:
Oui, on peut effectivement mettre «The signaux-faibles contributors» pour plus de généralité. 

Par contre, cela soulève le point qu'il manque une licence au dépôt. Je crois que par défaut, en l'absence d'une licence explicite sur un projet (même public), les auteurs en ont l'usage exclusif !  Je vois que @chrnin a utilisé une licence MIT sur les autres dépôt. Je peux l'ajouter par la même occasion dans cette PR.